### PR TITLE
chore(Layout): fix proptypes warning

### PR DESCRIPTION
**Pull request by jmfrancois**
array of children is not a react element

**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

Some warning appears:

* Layout: 

**What is the new behavior?**

Those warnings are not here anymore.

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://141.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://141.talend.surge.sh)

File: packages/components/src/Layout/Layout.component.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ Layout.propTypes = {
 	header: PropTypes.shape(AppHeaderBar.propTypes),
 	mode: PropTypes.oneOf(DISPLAY_MODES),
 	drawers: PropTypes.arrayOf(PropTypes.element),
-	children: PropTypes.element,
+	children: React.PropTypes.node,
```
Comment (acateland) [inline packages/components/src/Layout/Layout.component.js]: Proptypes.node will suffice

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://141.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://141.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://141.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://141.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://141.talend.surge.sh)

**Pull request merged**